Java/Mazes/src/Cell.java

```diff
@@ -0,0 +1,88 @@
+import java.util.ArrayList;
+
+class Cell {
+
+    private final int xCoor;
+    private final int yCoor;
+    private final ArrayList<Integer> availableDirections = new ArrayList<>();
+    private boolean visited;
+
+    /**
+     * Default Constructor
+     *
+     * @param xCoor The left/right coordinate
+     * @param yCoor The up/down coordinate
+     */
+    Cell(int xCoor, int yCoor) {
+        this.xCoor = xCoor;
+        this.yCoor = yCoor;
+    }
+
+    boolean isVisited() {
+        return !visited;
+    }
+
+    void setVisited() {
+        this.visited = true;
+    }
+
+    int getxCoor() {
+        return xCoor;
+    }
+
+    int getyCoor() {
+        return yCoor;
+    }
+
+
+
+    /**
+     * Checks that the number of available moves is greater than 0
+     *
+     * @return True is available populateAvailableDirections is greater than 0
```

Comment:
Change this comment to indeicate what true means, not the condition under when the return value is true.  I can see by reading the code that, if the size of avalibleDirections is greater than 0 it should return true.